### PR TITLE
Adds docs for new credentials

### PR DESCRIPTION
**Pull request by Joffcom**

### CODE REVIEW

**Comment by burivuhster** [comment]
https://linear.app/n8n/issue/AI-200/google-vertex-chat-model

File: docs/integrations/builtin/credentials/datadog.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/datadog/){:target=_blank .external-link} on n8n's website.
+
+
+## Using API Key
+
+To configure this credential, you'll need:
+
+- Host of your Datadog instance
+- API Key
+- App Key
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/datadog.md:27]:
```suggestion
- Your Datadog instance **Host**
- An **API Key**
- An **App Key**
```

File: docs/integrations/builtin/credentials/dfiriris.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/dfir-iris/){:target=_blank .external-link} on n8n's website.
+
+
+## Using API Key
+
+To configure this credential, you'll need:
+
+- API Key
+- Base URL
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/dfiriris.md:26]:
```suggestion
- An **API Key**: Refer to [DFIR-IRIS's API documentation](https://docs.dfir-iris.org/operations/api/){:target=_blank .external-link} for instructions on getting your API key.
- The **Base URL** of your DFIR-IRIS instance.
```

File: docs/integrations/builtin/credentials/filescan.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Refer to [Filescan's API documentation](https://www.filescan.io/api/docs){:target=_blank .external-link} for more information about authenticating with the service.
+
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/filescan/){:target=_blank .external-link} on n8n's website.
+
+
+## Using API key
+
+To configure this credential, you'll need:
+
+- API Key
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/filescan.md:25]:
```suggestion
- An **API Key**: Generate your API key from the API Key tab of your [profile settings](https://www.filescan.io/users/profile){:target=_blank .external-link}. Refer to the [Filescan FAQ](https://www.filescan.io/help/faq){:target=_blank .external-link} for more information.

Refer to 
```

File: docs/integrations/builtin/credentials/malcore.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Refer to [Malcore's API documentation](https://malcore.readme.io/reference/){:target=_blank .external-link} for more information about authenticating with the service.
+
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/malcore/){:target=_blank .external-link} on n8n's website.
+
+
+## Using API key
+
+To configure this credential, you'll need:
+
+- API Key
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/malcore.md:25]:
```suggestion
- An **API Key**: Get an API Key from your **Account > API**.

Refer to [Using the Malcore API](https://support.malcore.io/hc/en-au/articles/17711707070617-Using-the-Malcore-API){:target=_blank .external-link} for more information.
```

File: docs/integrations/builtin/credentials/elasticsecurity.md [hunk 5]
```diff
@@ -40,3 +41,13 @@ To configure this credential, you'll need:
 If you add a [custom endpoint alias](https://www.elastic.co/guide/en/cloud/current/ec-regional-deployment-aliases.html){:target=_blank .external-link} to a deployment, update your n8n credential **Base URL** with the new endpoint.
 ///
 
+## Using API key
+
+To configure this credential, you'll need:
+
+- An **API Key**: For the user account you log into Elasticsearch with.
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/elasticsecurity.md:48]:
```suggestion
- An **API Key**: For the user account you log into Elasticsearch with. Refer to Elasticsearch's [Create API key documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html){:target=_blank .external-link} for more information.
```

**Review by freakwriter — CHANGES_REQUESTED** [review]
@Joffcom I added some suggested commits here, mostly to bold the GUI fields in n8n, sometimes to add some additional links to the provider's docs. I also already committed some changes to the Vale checking so it would stop throwing errors about Datadog, DFIR-IRIS and Malcore. Should be good to go if you accept the suggested commits.

**Comment by Joffcom** [comment]
Thanks @freakwriter

File: docs/integrations/builtin/credentials/dynatrace.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+## Prerequisites
+
+Create a [Dynatrace](https://www.dynatrace.com/signup/){:target=_blank .external-link} account.
+
+## Related resources
+
+Refer to [Dynatrace's API documentation](https://docs.dynatrace.com/docs/dynatrace-api){:target=_blank .external-link} for more information about authenticating with the service.
+
+This is a credential-only node. Refer to [Custom API operations](/integrations/custom-operations/) to learn more. View [example workflows and related content](https://n8n.io/integrations/dynatrace/){:target=_blank .external-link} on n8n's website.
```
Comment (freakwriter) [inline docs/integrations/builtin/credentials/dynatrace.md:19]: @Joffcom the link to the marketing site integrations gave me a 404. Is this the correct link: https://n8n.io/integrations/dynatrace-api/ or is the one you have here correct and just pending-dev?

**Review by freakwriter — COMMENTED** [review]
Most of the links to marketing integration pages are 404ing, but I'm assuming that's because the nodes haven't actually been released yet. Dynatrace did seem like it was out there though, so I added a comment in case that's just the wrong URL. Let me know if it's not and I can approve as-is.

I added contentType: integration into the frontmatter for all of these, since they should have it but it seemed silly to ask you to do that.

File: docs/integrations/builtin/credentials/dynatrace.md (hunk 6 above)
Comment (Joffcom) [reply docs/integrations/builtin/credentials/dynatrace.md:19]: I didn't realise that was already there, I will get this updated thanks.

**Review by freakwriter — APPROVED** [review]
@Joffcom good to ship from my end!

**Pull request merged**